presentations/slides/itk-wasm-slides/info.json

```
@@ -0,0 +1,11 @@
+{
+    "title": "ITK-Wasm: Universal spatial analysis and visualization"
```

```suggestion
    "title": "ITK-Wasm: Universal spatial analysis and visualization",
```
Hi! Looks like this is missing a comma in order to be valid json.

---

@mepa thanks!